pyaerocom/io/read_eea_aqerep.py

```diff
@@ -235,7 +254,8 @@ def read_file(self, filename, vars_to_retrieve=None,
                     if idx in time_indexes:
                         # make the time string ISO compliant so that numpy can directly read it
                         # this is not very time string forgiving but fast
-                        data_dict[header[idx]][lineidx] = np.datetime64(rows[idx][0:10]+'T'+rows[idx][11:19]+rows[idx][20:])
+                        dtstr = rows[idx][0:10]+'T'+rows[idx][11:19]+rows[idx][20:]
```

Comment:
I did this in one line on purpose to save some time. This is run millions of times...